pkg/mtls/crypto.go

```diff
@@ -241,6 +249,12 @@ func createSigningPolicy() *config.Signing {
 	}
 }
 
+func createCrsSigningPolicy() *config.Signing {
+	return &config.Signing{
+		Default: createSigningProfile(crsProfileDefaultValidityPeriod, beforeGracePeriod),
```

Comment:
Cannot this one be a part of the `createSigningPolicy()` profiles at lines 245-247?